schemas/common/b2b-lead.schema.json

```diff
@@ -0,0 +1,98 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/b2b-lead",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "B2B Lead",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "This Data Type is used to capture B2B Lead specific attributes.",
+  "definitions": {
+    "b2b-lead": {
+      "properties": {
+        "xdm:ID": {
+          "title": "Lead Id",
+          "description": "Lead id.",
+          "type": "string"
+        },
+        "xdm:converted": {
+          "title": "Converted Indicator",
+          "description": "Indicates whether the Lead was successfully converted or not.",
+          "type": "boolean"
+        },
+        "xdm:convertedAccountID": {
+          "title": "Converted Account ID",
+          "description": "Account Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:convertedContactID": {
+          "title": "Converted Contact ID",
+          "description": "Contact Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:convertedDate": {
+          "title": "Converted Date",
+          "description": "Date of conversion if the lead was successfully converted.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:convertedOpportunityID": {
+          "title": "Converted Opportunity ID",
+          "description": "Opportunity Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:jigsawContactID": {
+          "title": "Jigsaw Contact Id",
+          "description": "Lead’s Data.com ID if available.",
+          "type": "string"
+        },
+        "xdm:lastTransferDate": {
+          "title": "Last Transfer Date",
+          "description": "Last transfer date.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:unReadByOwner": {
+          "title": "Read By Owner Indicator",
+          "description": "Read by owner indicator.",
+          "type": "boolean"
+        },
+        "xdm:source": {
+          "title": "Lead Source",
+          "description": "Field recording what source the lead originated from.",
+          "type": "string"
+        },
+        "xdm:organization": {
+          "title": "Organization Attributes",
+          "description": "Organization attributes if the lead is an organization.",
+          "$ref": "https://ns.adobe.com/xdm/common/organization"
+        },
+        "xdm:preferences": {
```

Comment:
See previous comment

Reply:
Generally the opt in/out flags are maintained under preferences for the entity under question. The idea for this name is also taken from the profile-preferences mixin which is an existing construct on XDM. We shouldn't give any opt-in/out name to this property as it could be an opt in/out for a certain channel for communication or general opt out for selling and sharing of the data. We will keep this generic enough.

Reply:
Given this "preferences" is of type optinout only, I agree that is should be named something like optInOutPreferences. If it was for general preferences that including many more flags and options other then just opting in and out then "preferences" may be okay.

Reply:
Will rename it to give it more semantic meaning.